include/wil/result_macros.h

```diff
@@ -1309,6 +1309,16 @@ namespace wil
             {
                 (void)g_pfnRtlDisownModuleHeapAllocation(::GetProcessHeap(), allocation);
             }
+            else if (!g_fetchedRtlDisownModuleHeapAllocation)
+            {
+                g_pfnRtlDisownModuleHeapAllocation = reinterpret_cast<decltype(g_pfnRtlDisownModuleHeapAllocation)>(::GetProcAddress(::GetModuleHandleW(L"ntdll.dll"), "RtlDisownModuleHeapAllocation"));
+                g_fetchedRtlDisownModuleHeapAllocation = true;
+
+                if (g_pfnRtlDisownModuleHeapAllocation)

```

Comment:
This logic does seem a little unnatural, but the _common_ case is that the pointer is set once the process starts, so I thought I'd leave that in place and put the boolean-check afterwards.

Reply:
@jonwis wondering as to when this change will arrive in OS2020 repo. Thanks.